src/spde_precision_matrices.jl

```diff
@@ -1,4 +1,16 @@
+"""
+    component_matrices(mesh::TriMesh, κ::real) -> (C::Matrix, C̃::Matrix, G::Matrix)
 
+From a triangulated mesh of a 2-dimensional domain, generate the finite element
+method matrices C, C̃, and G as described in Lindgren et al. 2011.
+
+# Arguments
+- `mesh::TriMesh`: a mesh from TriangleMesh.jl
+- `κ::Real`: Matérn smoothness parameter
```

Comment:
This should be "Matérn correlation decay rate parameter"

Reply:
Fixed below